nx-dev/ui-markdoc/src/lib/tags/cards.component.tsx

```diff
@@ -6,9 +6,17 @@ import {
 } from '@heroicons/react/24/outline';
 import { ReactNode } from 'react';
 
-export function Cards({ children }: { children: ReactNode }) {
+export function Cards({
+  cols = 2,
+  children,
+}: {
+  cols: number;
+  children: ReactNode;
+}) {
   return (
-    <div className="not-prose mt-8 grid grid-cols-1 gap-6 lg:grid-cols-2">
+    <div
+      className={`not-prose mt-8 grid grid-cols-1 gap-6 lg:grid-cols-${cols}`}
```

Comment:
This syntax is not supported by Tailwindcss, it only works for you because the somewhere in the codebase we are generating the classes you actually use. See here for more info: https://v2.tailwindcss.com/docs/just-in-time-mode#arbitrary-value-support

Please use a dictionary like syntax like this:
```typescript
const cols = {
  1: `lg:gird-cols-1`,
  2: `lg:gird-cols-2`,
  3: `lg:gird-cols-3`,
  4: `lg:gird-cols-4`,
};
```

Reply:
right 👍 . Makes sense. Will fix tomorrow. thanks